### PR TITLE
make spa default in SCREAM compsets

### PR DESCRIPTION
**Pull request by hassanbeydoun**

### CODE REVIEW

File: components/eam/cime_config/config_component.xml [hunk 1]
```diff
@@ -82,11 +82,11 @@
       <value compset="_EAM%ADIAB"     >-phys adiabatic</value>
 
       <!-- SCREAM configurations -->
-      <value compset="_EAM%SCREAM-LR">-phys default -shoc_sgs -microphys p3 -chem none -nlev 72 -rad rrtmgp -bc_dep_to_snow_updates</value>
-      <value compset="_EAM%SCREAM-HR">-phys default -shoc_sgs -microphys p3 -chem none -nlev 128 -rad rrtmgp -bc_dep_to_snow_updates</value>
+      <value compset="_EAM%SCREAM-LR">-phys default -shoc_sgs -microphys p3 -chem spa -nlev 72 -rad rrtmgp -bc_dep_to_snow_updates</value>
```
Comment (crterai) [inline components/eam/cime_config/config_component.xml:85]: So simple! Quick question though: will running with spa chemistry automatically figure out which input data files to run spa with? And are the data files in the inpudata repository on ALCF?

**Review by crterai — COMMENTED** [review]
This seems quite straightforward. Just have one question regarding the input data for spa.

**Review by brhillman — APPROVED** [review]
This looks good to me.

**Comment by PeterCaldwell** [comment]
@bartgol and @jgfouca - the v0 team would like this PR integrated. It changes our nightlies to actually test the configuration we run, but shouldn't have anything to do with the AT. Can I integrate it without testing on mappy, or do you want to manually test on mappy?

File: components/eam/cime_config/config_component.xml (hunk 1 above)
Comment (crterai) [reply components/eam/cime_config/config_component.xml:85]: Okay, confirmed that the input data file is on ALCF: 
`/lcrc/group/acme/public_html/inputdata/atm/cam/chem/spa/spa_mixing_ratio_data.nc`

**Review by crterai — APPROVED** [review]

**Comment by jgfouca** [comment]
Testing by hand now.

**Comment by jgfouca** [comment]
This branch was way behind current master so I did an upstream merge.

**Comment by jgfouca** [comment]
All PASS (expect for baseline diffs) on mappy. Merging.

**Pull request merged**